lib/index.ts

```diff
@@ -215,24 +215,19 @@ export default class Backburner {
   public run() {
     let [target, method, args] = parseArgs(...arguments);
 
-    let onError = getOnError(this.options);
-
     this.begin();
 
-    if (onError) {
-      try {
-        return method.apply(target, args);
-      } catch (error) {
+    try {
+      return method.apply(target, args);
+    } catch (error) {
```

Comment:
This existence of this catch will affect pause on caught exceptions even if `getOnError` is null. The original design with either a try/finally or a try/catch/finally was to ensure debugging ergonomics are good. This I believe regresses them

Reply:
@bekzod thoughts?